mobile/src/helpers/object/sort-by-date/sort-by-date.helper.ts

```diff
@@ -0,0 +1,22 @@
+import { compareAsc, compareDesc } from 'date-fns';
+
+type sortingType = 'desc' | 'asc';
+
+type sortItems<T> = {
+  items: T[];
+  key: keyof T;
+};
+
+const sortByDate = <T>(
```

Comment:
I think there is no need for such common helper. Let's just create a local helper similar to group-messages-by-date
```
const sortMessagesByDate = (messages) => {
  return messages.sort(a, b => newDate(a.createdAt) - newDate(b.createdAt));
};
```

Reply:
@revenets I`m so sorry for your time, we with @Liza-Veis had a long conversation about it, and decided to leave local sorter helper

Reply:
No problem!)